### DISCUSSION TITLE
能否给个文档如何本地运行addx？

### DISCUSSION THREAD

**Discussion by milton4code** [wgzhao/Addax#333] [Ideas]
感觉addx框架还是蛮不错的，尝试了多次在本地运行addx一直没有成功，能否给份文档指导一二，谢谢。

**Comment by grunt1024** [comment]
> 感觉addx框架还是蛮不错的，尝试了多次在本地运行addx一直没有成功，能否给份文档指导一二，谢谢。

我现在也要在本地调试，请问您现在解决了吗？如果解决了，能否告知本地调试方法，多谢。

**Comment by milton4code** [comment]
我还是没有成功运行起来，不过你可以参考下这个链接，或许你能有所收获
https://addax.readthedocs.io/zh/latest/debug.html#
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "wgzhao/Addax"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月6日(星期一) 上午9:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [wgzhao/Addax] 能否给个文档如何本地运行addx？ (#333)





  
感觉addx框架还是蛮不错的，尝试了多次在本地运行addx一直没有成功，能否给份文档指导一二，谢谢。
  
我现在也要在本地调试，请问您现在解决了吗？如果解决了，能否告知本地调试方法，多谢。
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by grunt1024** [reply]
多谢了，现在已经解决。
具体可以参考 [#336](https://github.com/wgzhao/Addax/discussions/336)
